docker-compose.yml changes

the file docker-compose.yml was changed yesterday and now we are getting this error 

COMPOSE_DOCKER_CLI_BUILD=1 docker-compose -f docker-compose.yml build db web proxy redis celery celery-beat tor
[+] Building 0.0s (2/2) FINISHED
 => [rengine_celery-beat internal] load build definition from Dockerfile                                                                                                                                    0.0s
 => => transferring dockerfile: 3.36kB                                                                                                                                                                      0.0s
 => [rengine_celery-beat internal] load .dockerignore                                                                                                                                                       0.0s
 => => transferring context: 2B                                                                                                                                                                             0.0s
failed to solve: dockerfile parse error on line 44: unknown instruction: <<<<<<<
make: *** [Makefile:25: build] Error 17

--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -112,7 +112,9 @@ services:
       - celery
       - celery-beat
     networks:
-      - rengine_network
+      rengine_network:
+        aliases:
+          - rengine
 
   tor:
     image: peterdavehello/tor-socks-proxy

--- a/web/Dockerfile
+++ b/web/Dockerfile
@@ -75,7 +75,7 @@ RUN GO111MODULE=on go install -v -v github.com/bp0lr/gauplus@latest
 
 RUN GO111MODULE=on go install -v github.com/jaeles-project/gospider@latest
 
-RUN go install -v github.com/OWASP/Amass/v3/...@latest
+RUN go install -v github.com/owasp-amass/amass/v3/...@latest
 
 RUN go install -v github.com/ffuf/ffuf@latest
 

--- a/web/fixtures/external_tools.yaml
+++ b/web/fixtures/external_tools.yaml
@@ -96,16 +96,16 @@
 - model: scanEngine.installedexternaltool
   pk: 6
   fields:
-    logo_url: https://raw.githubusercontent.com/OWASP/Amass/master/images/amass_logo.png
+    logo_url: https://raw.githubusercontent.com/owasp-amass/amass/master/images/amass_logo.png
     name: amass
     description: The OWASP Amass Project performs network mapping of attack surfaces
       and external asset discovery using open source information gathering and active
       reconnaissance techniques.
-    github_url: https://github.com/OWASP/Amass
-    license_url: https://github.com/OWASP/Amass/blob/master/LICENSE
+    github_url: https://github.com/owasp-amass/amass
+    license_url: https://github.com/owasp-amass/amass/blob/master/LICENSE
     version_lookup_command: amass -version
-    update_command: go install -v github.com/OWASP/Amass/v3/...@latest
-    install_command: go install -v github.com/OWASP/Amass/v3/...@latest
+    update_command: go install -v github.com/owasp-amass/amass/v3/...@latest
+    install_command: go install -v github.com/owasp-amass/amass/v3/...@latest
     version_match_regex: v(\d+\.)?(\d+\.)?(\*|\d+)
     is_default: true
     is_subdomain_gathering: true

--- a/web/scanEngine/templates/scanEngine/settings/tool.html
+++ b/web/scanEngine/templates/scanEngine/settings/tool.html
@@ -64,7 +64,7 @@ Settings, API Keys & Tool Settings
       <div class="card-body">
         <h4 class="header-title">Amass</h4>
         <p>
-          This section lets you modify <a href="https://github.com/OWASP/Amass/" class="text-primary" target="_blank">Amass</a> config file. You can find more information about the <a href="https://github.com/OWASP/Amass/blob/master/examples/config.ini" class="text-primary" target="_blank">Amass config here.</a><br>
+          This section lets you modify <a href="https://github.com/owasp-amass/amass/" class="text-primary" target="_blank">Amass</a> config file. You can find more information about the <a href="https://github.com/owasp-amass/amass/blob/master/examples/config.ini" class="text-primary" target="_blank">Amass config here.</a><br>
           <span class="text-danger">Please note that only YAML config is supported.</span>
         </p>
         <form method="post" enctype="multipart/form-data" id="amass-config-form">

